pkg/kubectl/cmd/set/helper.go

```diff
@@ -118,14 +118,14 @@ type Patch struct {
 	Patch  []byte
 }
 
-// patchFn is a function type that accepts an info object and returns a byte slice.
-// Implementations of patchFn should update the object and return it encoded.
-type patchFn func(*resource.Info) ([]byte, error)
+// PatchFn is a function type that accepts an info object and returns a byte slice.
+// Implementations of PatchFn should update the object and return it encoded.
+type PatchFn func(*resource.Info) ([]byte, error)
```

Comment:
does anything you pass here actually use a resource info?  If not, don't make the shim, just fix this function.

Reply:
I have only seen it used in warnings: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/set/set_env.go#L330 and print statements https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/set/set_env.go#L345